index.js

```diff
@@ -263,10 +263,6 @@ app.get('/install', function(req, res) {
   res.redirect('http://start.tessel.io');
 });
 
-app.get('/docs', function(req, res) {
```

Comment:
I moved this down to be with the rest of the docs redirects. No idea why it was in this order before (we srsly need a repo-wide cleanup for this thing to make sense)